tests/models/transformers/test_models_dit_transformer2d.py

```diff
@@ -34,6 +34,7 @@
 class DiTTransformer2DModelTests(ModelTesterMixin, unittest.TestCase):
     model_class = DiTTransformer2DModel
     main_input_name = "hidden_states"
+    uses_custom_attn_processor = False
```

Comment:
We could just default it to `False` in `ModelTesterMixin `, no?

Reply:
Good point. I'll update.